src/modules/navigator/navigator_mode.cpp

```diff
@@ -48,12 +48,19 @@ NavigatorMode::NavigatorMode(Navigator *navigator) :
 	/* set initial mission items */
 	on_inactivation();
 	on_inactive();
+
+	/* invalidate position setpoint by default */
+	_navigator->get_position_setpoint_triplet()->current.valid = false;
```

Comment:
Because when I was in Gazebo simulation, I found that it was not running here.

Reply:
I'm against this change. It's scary and I'm not sure we know all of the effects of this.

Reply:
@julianoes  You are right, but can I just delete it instead of moving here?
Because it will not be run here(https://github.com/PX4/Firmware/pull/13740/files#diff-ca6b0e7090d881629e9e3cd91610fb09L95), it is invalid code
thanks review :smiley:

Reply:
done here: https://github.com/PX4/Firmware/pull/13740/commits/5aa037a00815ad3934ee8d9908482340f7151ef3

FYI : @sfuhrer